units/systemd-machine-id-commit.service

```diff
@@ -21,4 +21,4 @@ ConditionPathIsMountPoint=/etc/machine-id
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=systemd-machine-id-setup --commit
-TimeoutSec=30s
+TimeoutSec=90s
```

Comment:
so it seems various other of our Type=onsehot units set 90s too, and machine-id-commit is the exception, hence makes sense to normalize this.

That said, there's another outlier: systemd-rfkill.service

If you change the value here, maybe adjust that other service to 90s too? just to be less surprising.